RetsTalk/RetsTalk/User/Model/UserSettingManager.swift

```diff
@@ -7,43 +7,42 @@
 
 import Combine
 
-final class UserSettingManager: UserSettingManageable, @unchecked Sendable {
-    private var userData: UserData {
-        didSet { userDataSubject.send(userData) }
-    }
-    private(set) var userDataSubject: CurrentValueSubject<UserData, Never>
+final class UserSettingManager: UserSettingManageable, @unchecked Sendable, ObservableObject {
+    @Published var userData: UserData = .init(dictionary: [:])
     private let userDataStorage: Persistable
     
     // MARK: Init method
     
-    init(userData: UserData, persistent: Persistable) {
-        self.userData = userData
-        userDataSubject = CurrentValueSubject(userData)
+    init(persistent: Persistable) {
```

Comment:
```suggestion
    init(userDataStorage: Persistable) {
```
다른 코드와의 일관성 + 가독성을 위해서 네이밍을 변경해도 좋을 것 같네요!